tests/test_io.py

```diff
@@ -441,6 +480,7 @@ def test_load_scan_get_data(
     scan = request.getfixturevalue(load_scan_object)
     scan.get_data()
     assert len(scan.img_dict) == length
+    print(scan.img_dict)
```

Comment:
Errant debug `print()`.

Surprised `ruff` doesn't pick this up (see [print (T201)](https://docs.astral.sh/ruff/rules/print/)) either locally 
when running the `pre-commit` hook or here on when `pre-commit.ci` runs.

I'll look into that as we've had to explicitly disable this in `tests/test_processing.py` where `pytest.regtest` requires a `print()` statement to write the CSV files for the regression tests.